### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by tipassa**

### CODE REVIEW

File: Periscope_Web_Client.user.js [hunk 1]
```diff
@@ -1866,7 +1864,7 @@ function addUserContextMenu(node, id, username) {
                     user_id: id
                 });
             }))
-            .append('<div data-clipboard-text="https://periscope.tv/' + username + '">Copy profile URL</div>' +
+            .append('<div data-clipboard-text="' + username + '">Copy username</div>' +
```
Comment (gitnew2018) [inline Periscope_Web_Client.user.js]: Maybe someone liked the original function. Justify it.

File: style.css [hunk 2]
```diff
@@ -97,6 +97,28 @@ a, .edit {
     cursor: pointer;
 }
 
+a.broadcastTitle {
```
Comment (gitnew2018) [inline style.css]: what's the point of that? its <a> so the cursor i already pointer and there is no text decoration.

File: style.css [hunk 3]
```diff
@@ -97,6 +97,28 @@ a, .edit {
...
+    color: #039be5;
+    text-decoration: none;
+    cursor: pointer;
+}
+
+a.downloadGet {
```
Comment (gitnew2018) [inline style.css]: these two ok, except unnecessary cursor, decoration and display.

File: style.css [hunk 4]
```diff
@@ -781,10 +826,18 @@ span.sProducer {
 
 .linkLive {
     color:#ED4D4D;
+	text-decoration: none;
```
Comment (gitnew2018) [inline style.css]: I don't really like this change, imho it looks worse.

File: Periscope_Web_Client.user.js [hunk 5]
```diff
@@ -129,7 +129,7 @@ function lazyLoad(parent) {
             right.find('img[lazysrc]:visible').each(function () {
                 var el = $(this);
                 var top = el.offset().top;
-                if (scrollTop < top + el.height() + 100 && scrollTop + windowHeight + 100 > top) {  // 100 is handicap
+                if (scrollTop < top + el.height() + 1400 && scrollTop + windowHeight + 1400 > top) {  // 100 is handicap
```
[the remark this answers is not in the document]
Comment (tipassa) [reply Periscope_Web_Client.user.js]: Page loading seemed very slow.

File: Periscope_Web_Client.user.js [hunk 6]
```diff
@@ -753,7 +751,7 @@ Top: function () {
 
     var refreshOnLoad = $('<label/>Refresh on load</label>').prepend(refreshOnLoadBtn);
 
-    var TopObj = $('<div id="Top"/>').append(langDt, button, refreshOnLoad, featured, ranked);
+    var TopObj = $('<div id="Top"/>').append(langDt, button, refreshOnLoad, ranked);
```
[the remark this answers is not in the document]
Comment (tipassa) [reply Periscope_Web_Client.user.js]: I made this fix for myself. This item seemed redundant.

File: Periscope_Web_Client.user.js (hunk 1 above)
Comment (tipassa) [reply Periscope_Web_Client.user.js]: You can put both. Added for convenience only.

File: style.css (hunk 2 above)
Comment (tipassa) [reply style.css]: Yes, this item is superfluous.

File: style.css (hunk 3 above)
Comment (tipassa) [reply style.css]: I agree

File: style.css (hunk 4 above)
Comment (tipassa) [reply style.css]: I agree) suggested as your option ... Your design is probably better.

**Comment by tipassa** [comment]
It would be nice if the size of the preview could be customized, since it’s very small.

**Comment by tipassa** [comment]
some mistakes due to ignorance. so do not judge strictly)

**Comment by gitnew2018** [comment]
I'm not judging you, only trying to calmly think which changes are good for the project and which aren't. I' make many mistakes myself. It's like 1 step forward 2 steps back for me.

I'm going to close this PR and add good changes into next commit later.

I'm not opposed to option for bigger thumbs but it has to be done right. It's also very low priority.

**Pull request closed without merging**